cmd/kubebuilder/create/resource/run.go

```diff
@@ -74,14 +76,23 @@ func RunCreateResource(cmd *cobra.Command, args []string) {
 
 	cr := util.GetCopyright(createutil.Copyright)
 
-	fmt.Printf("Creating API files for you to edit...\n")
-	createGroup(cr)
-	createVersion(cr)
-	createResource(cr)
-	if generate {
-		fmt.Printf("Generating code for new resource...  " +
-			"Regenerate after editing resources files by running `kubebuilder build generated`.\n")
-		generatecmd.RunGenerate(cmd, args)
+	if coreControllerOnly {
+		fmt.Printf("Creating controller for kubernetes core types ...\n")
+		createCoreController(cr)
+		if generate {
+			args = append(args, "core-controller-only")
+			generatecmd.RunGenerate(cmd, args)
+		}
```

Comment:
hmm....This doesn't super intuitive that controller will be generated when I run command `create resource` command for `controller-only` project. Need to think more.

Reply:
We can do this at the `init` step itself for `controller-only` project ? or may be I am missing something.